lib/rubocop/cop/rspec/describe_method.rb

```diff
@@ -3,32 +3,123 @@
 module RuboCop
   module Cop
     module RSpec
-      # Checks that the second argument to `describe` specifies a method.
+      # Checks that arguments and sub-describes of `describe` specify a method.
       #
       # @example
       #   # bad
-      #   describe MyClass, 'do something' do
+      #   describe MyClass do
+      #     describe 'flowers' do; end
+      #   end
+      #
+      #   describe MyClass, 'flowers' do
       #   end
       #
       #   # good
+      #   describe MyClass do
+      #     describe '#my_instance_method' do; end
+      #   end
+      #
+      #   describe MyClass do
+      #     describe '.my_class_method' do; end
+      #   end
+      #
       #   describe MyClass, '#my_instance_method' do
       #   end
       #
       #   describe MyClass, '.my_class_method' do
       #   end
+      #
+      # @example configuration
+      #
+      #   # .rubocop.yml
+      #   RSpec/DescribeMethod:
+      #     IgnoredDescribes:
```

Comment:
We have a couple of predefined describe blocks that are added to internal framework component tests, it looks pretty readable and I wanted to preserve the option of doing such things. It's kind of from my own personal need, so it might not be used very often, but then again - this shouldn't hurt performance too much and is quite isolated.